packages/modules/cra-template-web-viewer/template/src/Context.tsx

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { WebAuthorizationOptions } from "@itwin/web-viewer-react";
+import React from "react";
+
+export interface IModelJsConfig {
```

Comment:
let's remove "imodeljs..." from these names. Technically, there is no more imodeljs so, even though its referenced elsewhere, we don't want to add it in new places. ViewerContext might be more meaningful and will help to obscure the underlying tech. Also should rename the file to match the name of the context.

Reply:
Renamed to ViewerContext.